besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -372,4 +373,18 @@ protected EthProtocolManager createEthProtocolManager(
 
   protected abstract PluginServiceFactory createAdditionalPluginServices(
       final Blockchain blockchain);
+
+  private List<Long> gatherForks() {
+    // todo: may need to check what protocol version is being used. (i.e. if less than x return null
+    // to disable this check)
+    List<Long> listb = new ArrayList<>();
+    Map<String, Object> values = genesisConfig.getConfigOptions(genesisConfigOverrides).asMap();
+    values.forEach(
+        (x, y) -> {
+          if (y instanceof Long && !x.equals("chainId")) {
+            listb.add((Long) y);
+          }
+        });
+    return listb;
```

Comment:
This feels like it's leaking implementation details. What do you think about having a method on the genesis config that will provide the list of fork?

Reply:
Yeah, it does.  I moved it over to the config and I think it may be better.